includes/class-amp-post-template.php

```diff
@@ -254,22 +255,27 @@ private function build_post_data() {
 		) );
 
 		$metadata = array(
-			'@context' => 'http://schema.org',
-			'@type' => 'BlogPosting',
+			'@context'         => 'http://schema.org',
 			'mainEntityOfPage' => $this->get( 'canonical_url' ),
-			'publisher' => array(
+			'headline'         => $post_title,
+			'publisher'        => array(
 				'@type' => 'Organization',
-				'name' => $this->get( 'blog_name' ),
-			),
-			'headline' => $post_title,
-			'datePublished' => date( 'c', $post_publish_timestamp ),
```

Comment:
These Schema.org properties are also relevant to `WebPage` entities: http://schema.org/WebPage

So I don't see why we need to limit them just to posts.

Reply:
True, but this is just in regards to the template rendering of the data. We're talking about metadata here. It should be up to the application to decide whether or not to show the `datePublished` if the `type` is `WebPage`. In other words, the WP REST API for pages includes the published date, and that I think is a better parallel to the Schema.org metadata here.

Reply:
Sounds good, I am onboard 😄